src/font/font_face.rs

```diff
@@ -70,11 +71,10 @@ impl FontFace {
     }
 
     pub fn toy_get_family(&self) -> Option<String> {
-        unsafe {
-            let family_name = ffi::cairo_toy_font_face_get_family(self.to_raw_none());
-            if family_name.is_null() { None }
-            else { Some(String::from_utf8_lossy(CStr::from_ptr(family_name).to_bytes()).into_owned()) }
-        }
+        let family_name = unsafe {
+            ffi::cairo_toy_font_face_get_family(self.to_raw_none())
+        };
+        to_optional_string(family_name)
```

Comment:
This needs to be inside the unsafe block, and `to_optional_string itself` needs to be an unsafe fn.